portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,7 +31,7 @@
 public class DataServlet extends HttpServlet { 
   
     // ArrayList<String> words = new ArrayList<String>();
```

Comment:
I would go ahead and delete these two commented out lines to keep things tidy.

Reply:
Will do!